themes/bootstrap3/js/common.js

```diff
@@ -649,6 +649,50 @@ function setupQRCodeLinks(_container) {
   });
 }
 
+// Useful function to delay callbacks, e.g. when using a keyup event
+// to detect when the user stops typing.
+// See also: https://stackoverflow.com/questions/1909441/how-to-delay-the-keyup-handler-until-the-user-stops-typing
+function getFacetListContentKeyupCallback() {
+  var timeout = null;
+  $('#ajax_contains').on('keyup', function onKeyupChangeFacetList() {
+    clearTimeout(timeout);
+    timeout = setTimeout(function onKeyupTimeout() {
+      getFacetListContent();
+    }, 500);
+  });
+}
+
+function getFacetListContent() {
+  let facet = $('#ajax_facet').val();
+  var url = VuFind.path + "/AJAX/JSON?q=sta&method=getFacetListContent&facet=" + facet;
```

Comment:
nitpick: can we change this to `let` for consistency?

Reply:
done